### ISSUE TITLE
Unable to create pull requests

### ISSUE THREAD

**Issue by intel352** [gitbutlerapp/gitbutler#5779]
### Version

0.14.3

### Operating System

macOS

### Distribution Method

dmg (Apple Silicon)

### Describe the issue

Launching has this error:

```
791c3c29d71a47699444192fc95758f6: Not Found - https://docs.github.com/rest/pulls/pulls#list-pull-requests w@tauri://localhost/_app/immutable/chunks/public.D_6bGMXX.js:6:6011 @tauri://localhost/_app/immutable/chunks/public.D_6bGMXX.js:6:8483
```

Attempting to create a pull request has this error:
```
Not Found - https://docs.github.com/rest/pulls/pulls#create-a-pull-request
```

Quite strange. I'm doing this on a new macbook. The last macbook I had was of course running an older release of GitButler, but I'd been able to create and manage PRs from the app.
Something about either the configuration with the new macbook, or else the version of Gitbutler that is active, is leading me into errors that I've not seen before.

I recreated the Github integration authentication. I enabled git signing via ssh, I have git configured to auth via ssh correctly and GitButler tests correctly. I'm not sure what else I need to check.

**Comment by Byron** [comment]
Thanks a lot for reporting!

It's interesting that it gets a 404 - Not Found - and the URL looks strange as well. Also, for me it just worked without issues.

CC @estib-vega - maybe this seems familiar.

**Comment by estib-vega** [comment]
Thanks for tagging me @Byron :)

Hey there @intel352.
Thanks for reporting and sorry for the inconvenience!

I'm also having issues reproducing this. 
Is there any thing interesting in the frontend logs?
You should be able to peek into the logs and network calls by opening the browser dev tools `option` + `command` + `i`.

**Comment by intel352** [comment]
I'll check on the logs later today, thanks!

**Comment by intel352** [comment]
Hmm, it's attempting to hit `https://api.github.com/repos/%2FCrisisTextLine/terraform-root-modules/pulls`.
Not sure where the `%2F` came from, seeing if I can find it

**Comment by intel352** [comment]
Okay, so the `.github/gitbutler/` folder has an unexpected value:

```
jlangevin@DCF4NW992W gitbutler % ag Crisis
virtual_branches.toml
4:remoteUrl = "git@github.com:/CrisisTextLine/terraform-root-modules.git"
```

This has to be where the `%2F` is coming from. Not sure how it originated.

Feel free to close if you don't see a need to check for weird inputs like this :)

**Comment by Byron** [comment]
Indeed, the URL has a leading `/` and that is encoded with `%2F` as part of the organisation/user name.
I assume after removing it, it worked.

Can you check the output of `git remote -v`? My guess is that it already contains the slash and somehow Git can deal with it. Maybe it's no issue for SSH-style URLs and it's something we have to specifically handle when processing the URL.

Thanks again.

**Comment by intel352** [comment]
You're correct, the remote listing has the leading slash. I didn't specify that. In the past I've typically cloned via cli by copying the clone command from Github, but on this laptop I believe I let Intellij IDEA clone via it's Github-aware project import.

Other git actions don't have a problem either, it's just when GitButler attempts to keep and encode that `/` value when calling Github's APIs


Sent with Proton Mail secure email.

**Comment by Byron** [comment]
Maybe the problem is due to not 100% correct handling of URLs in the frontend? Maybe @estib-vega can take a look - ideally that's perfectly testable. The expectation would be this works as some tools seem to be able to produce such partially illformed (but working) URLs.

Alternatively to just making it work, we could of course also show an error that clearly states what the problem is, and how to solve it.

**Comment by PavelLaptev** [comment]
@Byron it wasn’t the UI, it's a backend/data parsing issue.

**Comment by Byron** [comment]
Thanks.

Also, given the age of the issue and the rate of change, I think we can close it.
